modules/CmpdReg/src/client/custom/configuration.json

```diff
@@ -1,112 +1,131 @@
 {
-    "serverConnection": {
-        "connectToServer": true,
-        "baseServerURL": "/cmpdreg/",
-        "logoutURL": "/logout",
-        "acasURL": "http://localhost:8080/acas/api/v1/"
-    },
-    "clientUILabels": {
-        "corpNameLabel": "CMPD Number",
-        "applicationNameForTitleBar": "ACAS Compound Registration",
-        "commonNameLabel": "Original ID:"
-    },
+	"serverConnection": {
+		"connectToServer": true,
+		"baseServerURL": "/cmpdreg/",
+		"logoutURL": "/logout",
+		"acasURL": "http://localhost:8080/acas/api/v1/",
+		"acasAppURL": "http://localhost:3001/api/"		 
+	},
+	"clientUILabels": {
+		"corpNameLabel": "CMPD Number",
+		"applicationNameForTitleBar": "ACAS Compound Registration",
+		"commonNameLabel": "Original ID:"
```

Comment:
I think "Common Name" would be better for default

Reply:
The actual UI field that uses this config was taken out when we added Aliases, so it is a dead config

Reply:
ok then